src/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -168,8 +168,10 @@
         </service>
         <service id="ApiPlatform\Api\IriConverterInterface" alias="api_platform.symfony.iri_converter" />
 
-        <service id="api_platform.error_listener" class="ApiPlatform\Symfony\EventListener\ErrorListener" parent="exception_listener">
```

Comment:
The ErrorListener still extends the Symfony one: https://github.com/api-platform/core/blob/main/src/Symfony/EventListener/ErrorListener.php

Reply:
this was changed on 2.7